### ISSUE TITLE
Incomplete extension with v0.4.1

### ISSUE THREAD

**Issue by daiyam** [HaaLeo/publish-vscode-extension#13]
Hi @HaaLeo,

I'm using your action to publish the extension [Explicit Folding](https://github.com/zokugun/vscode-explicit-folding).
Yesterday, (June 24), the action ran with the version v0.4.1 but the resulted extension was mostly empty (just the files in the root).

With v0.4.1
- the run: https://github.com/zokugun/vscode-explicit-folding/runs/2907374131?check_suite_focus=true
- the result: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/zokugun/vsextensions/explicit-folding/0.14.5/vspackage

With v0.4.0
- the run: https://github.com/zokugun/vscode-explicit-folding/runs/2875328422?check_suite_focus=true
- the result: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/zokugun/vsextensions/explicit-folding/0.14.4/vspackage

I'm using [.vscodeignore](https://github.com/zokugun/vscode-explicit-folding/blob/master/.vscodeignore) to control which files to include.

**Comment by HaaLeo** [comment]
Hi @daiyam thx for opening this issue. Can you try to explicitly use v0.4.0 for a new release and check whether this solves the issue? Then we are sure this is caused by the new version of the action

**Comment by daiyam** [comment]
Tested with https://open-vsx.org/extension/zokugun/explicit-folding (VS Marketplace was done manually)

`v0.4.1`:
```
Run HaaLeo/publish-vscode-extension@v0
Specifically, running GitHub Action HaaLeo/publish-vscode-extension@v0.4.1.
Package the Extension
  Start packaging the extension.
  Executing prepublish script 'npm run vscode:prepublish'...
  
  > explicit-folding@0.14.5 vscode:prepublish /home/runner/work/vscode-explicit-folding/vscode-explicit-folding
  > npm run compile
  
  
  > explicit-folding@0.14.5 compile /home/runner/work/vscode-explicit-folding/vscode-explicit-folding
  > tsc -p ./
  
   DONE  Packaged: explicit-folding-0.14.5.vsix (7 files, 11.04KB)
```

`v0.4.0`:
```
Run HaaLeo/publish-vscode-extension@v0.4.0
Specifically, running GitHub Action HaaLeo/publish-vscode-extension@v0.4.0.
Package the Extension
  Start packaging the extension.
  Executing prepublish script 'npm run vscode:prepublish'...
  
  > explicit-folding@0.14.6 vscode:prepublish /home/runner/work/vscode-explicit-folding/vscode-explicit-folding
  > npm run compile
  
  
  > explicit-folding@0.14.6 compile /home/runner/work/vscode-explicit-folding/vscode-explicit-folding
  > tsc -p ./
  
   DONE  Packaged: explicit-folding-0.14.6.vsix (25 files, 36.46KB)
```

- `v0.4.1`: Packaged: explicit-folding-0.14.5.vsix (7 files, 11.04KB)
- `v0.4.0`: Packaged: explicit-folding-0.14.6.vsix (25 files, 36.46KB)

**Comment by HaaLeo** [comment]
Turns out this is bug is caused by using [vsce](https://github.com/microsoft/vscode-vsce) ≥ `v1.92.0`. In that release they changed the behavior of the `.vscodeignore` file. For reference and a possible fix see microsoft/vscode-vsce#576

Unfortunately, I was not aware of that breaking change before I released v0.4.1 which upgraded vsce to `v0.94.0`. Sorry for that. However, I do not plan to rollback that behavior to keep the action alligned with vsce.

**Comment by HaaLeo** [comment]
Looking at microsoft/vscode-vsce#588. I will soon release `v0.4.2`  which will use vsce `v0.95.0` to revert the changes with the `.vscodeignore` file.
I hope to get this done this weekend. 🤞🏼

**Comment by daiyam** [comment]
Great! Thanks.

**Comment by HaaLeo** [comment]
Released with `v0.4.2`. Now the `v0` tag should work again as expected.

**Comment by daiyam** [comment]
Confirmed! The `v0.4.2` is fixing the issue. Thx
